src/env_properties.h

```diff
@@ -71,6 +71,9 @@
   V(__dirname_string, "__dirname")                                             \
   V(ack_string, "ack")                                                         \
   V(address_string, "address")                                                 \
+  V(total_pages_string, "totalPages")                                          \
+  V(remaining_pages_string, "remainingPages")                                  \
+  V(progress_string, "progress")                                               \
```

Comment:
Can you please place these alphabetically in the list?

Reply:
Done